### PR TITLE
NPM Release

### PR DESCRIPTION
**Pull request by zampino**
* adjust for NPM release
* update usage in README and landing page
* add squint build snapshot with import-from-CDN demo

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -53,18 +94,14 @@ jobs:
...
           cli: '1.10.3.943'
+
       - name: 🧶 Yarn Install
         run: yarn install
 
       - name: 🧶 Yarn Build
         run: |
           yarn build
 
-      - name: 🧶 Squint build
```
Comment (borkdude) [inline .github/workflows/main.yml:63]: Why was this removed?

File: README.md [hunk 2]
```diff
@@ -35,6 +35,25 @@ Enabling a decent Clojure/Script editor experience in the browser. Built for and
 {:deps {io.github.nextjournal/clojure-mode {:git/sha "<SHA>"}}}
 ```
 
+### Use if from [NPM](https://www.npmjs.com/package/@nextjournal/clojure-mode)
```
Comment (borkdude) [inline README.md:38]: Use if?

**Comment by sstraust** [comment]
I tried using this today, and I'm getting the error:

(!) Plugin node-resolve: Could not resolve import "@nextjournal/clojure-mode" in /Users/sam/sstraust/newsViewer/newsjs/resources/codemirror_editor.js using exports defined in /Users/sam/sstraust/newsViewer/newsjs/resources/node_modules/@nextjournal/clojure-mode/package.json. Invalid mapping: "dist/nextjournal/clojure_mode.mjs".


(I'm new in this area, so I could be missing something obvious)

**Comment by borkdude** [comment]
@sstraust Is it possible for you to put out a github repo with a repro of your problem?

**Comment by borkdude** [comment]
Perhaps this is the bug:

https://github.com/rollup/plugins/issues/886#issuecomment-847865309

@sstraust Could you try editing `node_modules/@nextjournal/clojure-mode/package.json and edit the `"exports"` field by adding `./` in front of `dist`, to see if this fixes it?

**Comment by sstraust** [comment]
yes! adding ./ in the suggested place removed the error warning from my build

**Comment by mk** [comment]
I've published [0.2.0 on npm](https://www.npmjs.com/package/@nextjournal/clojure-mode/v/0.2.0). Please let us know if that resolves your issue, thanks!

**Comment by sstraust** [comment]
yep! it now works for me!

(I edited package.json to use "^0.2.0", then ran:
`$ rm -r node_modules/
  $ npm install
  $ npx rollup codemirror_editor.js -f iife -o dist/cm6.bundle.js -p @rollup/plugin-node-resolve --output.name cm6`)

compilation ran with no warnings, and I am able to see the editor in my browser.
